main.py

```diff
@@ -372,35 +373,47 @@ def user_token(self, userid='core'):
     
     def check_docker(self):
         '''Check docker services status'''
-        dockercloud.user = self.config['DOCKERCLOUD_USER']
-        dockercloud.apikey = self.config['DOCKERCLOUD_APIKEY']
-        dockercloud.namespace = self.config['DOCKERCLOUD_NAMESPACE']
-        d = dockercloud.Service.list()
-        for service in d:
-            print(service.name + " : " + service.state)
-            
+        stack = "docker-cloud stack inspect datahub-production"
+        try:
+            process= subprocess.Popen(stack.split(), stdout=subprocess.PIPE)
+            output, error = process.communicate()
+            output = output.split(',')
+            print(output[5])
+        except:
+            print("Command not found")
+        
     def check_apis(self):
-        ''' Check API call for all services'''
+        '''Check API call for all services'''
         valid_token = self.config['JWT_TOKEN']
         invalid_token = 'ivalid_token'
-        api_base_url = 'http://' + self.config['DOMAIN_API'] 
+        api_base_url = 'http://api.' + self.config['DOMAIN_BASE'] 
```

Comment:
@zelima I could not automate this, since production_api = `api.datahub.io` and testing_api = `api-testing.datahub.io`. That's why I hardcoded production_api. Any idea how I can generate using env.var?